### PR TITLE
location for "level" parameter

### PR DESCRIPTION
**Pull request by Zatho**

### CODE REVIEW

**Comment by vulder** [comment]
hmm, regarding the model, we should not create a new one with the locations but update the existing one.

**Comment by ChristianKaltenecker** [comment]
The alternative group `level` could also be represented by a numeric feature with a numeric range of 1 to 9. 
The representation as a single numeric feature is also better for the feature location since the numeric feature level should only have a single code location (i.e., a single line where the numeric value for level is parsed).

**Comment by Zatho** [comment]
Yeah, found that very odd in the Model as well.I will compress that feature into one with a range of 1 to 9,if you could send me a quick example what the XML tag Looks like for thatand then I will instead of creating a new file replace the old one as Florian suggested. Von: ChristianKalteneckerGesendet: Donnerstag, 30. September 2021 17:41An: se-sic/ConfigurableSystemsCc: Zatho; AuthorBetreff: Re: [se-sic/ConfigurableSystems] location for "level" parameter (#2) The alternative group level could also be represented by a numeric feature with a numeric range of 1 to 9.The representation as a single numeric feature is also better for the feature location since the numeric feature level should only have a single code location (i.e., a single line where the numeric value for level is parsed).—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ChristianKaltenecker** [comment]
See [brotli](https://github.com/se-sic/ConfigurableSystems/blob/master/brotli/FeatureModel.xml) for example.
In the tag `numericOptions` are different numeric configuration options with a minumum value, a maximum value and a step function.

**Comment by Zatho** [comment]
So ich hab compressionLevel dann mal zu einer numericOption gemacht und das FeatureModel.xml direkt angepasst statt ein extra file zu erstellen.https://github.com/se-sic/ConfigurableSystems/pull/2/commits/580e303e6564065b04a0e5202e991be74425683b  Von: ChristianKalteneckerGesendet: Donnerstag, 30. September 2021 19:16An: se-sic/ConfigurableSystemsCc: Zatho; AuthorBetreff: Re: [se-sic/ConfigurableSystems] location for "level" parameter (#2) See brotli for example.In the tag numericOptions are different numeric configuration options with a minumum value, a maximum value and a step function.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

File: lrzip/FeatureModel.xml [hunk 1]
```diff
@@ -356,10 +160,25 @@
       <children />
       <impliedOptions />
       <excludedOptions />
-      <optional>True</optional>
-    </configurationOption>
-  </binaryOptions>
-  <numericOptions />
+      <optional>False</optional>
```
Comment (ChristianKaltenecker) [inline lrzip/FeatureModel.xml]: FYI: numeric options currently can not be declared optional or not (i.e., numeric configuration options are always there). This is a flaw in the SPL Conqueror format.

Other than that, it looks good to me.

**Review by ChristianKaltenecker — APPROVED** [review]

File: lrzip/FeatureModel.xml (hunk 1 above)
Comment (Zatho) [reply lrzip/FeatureModel.xml]: Alright, I'll throw that out then

**Review by vulder — APPROVED** [review]

**Pull request merged**